code/modules/paperwork/ticketmachine.dm

```diff
@@ -21,6 +21,7 @@
 	var/list/ticket_holders = list()
 	var/list/tickets = list()
 	var/id = 1
+	var/disabled = FALSE //used to ID card disable/enable ticket dispensing
```

Comment:
Use positive words for variables. They are much less confusing when you inevitably do negation.
Consider the extreme of `if(!disabled != FALSE)`, even that would be nicer as `if(enabled != FALSE)` (even though it's still bad).
For a less extreme and more fair example, consider `if(!enabled)` vs `if(!disabled)` and see which one is easier to understand. Yes, the difference is kinda small, but it's there

Reply:
I changed variable to isDisabled, thus eliminating completely all and any possible confusion with negation. 100%

Reply:
No, you're missing the point.
Disabled, is, semantically dis+abled = not enabled. So now, when you want to see if it's enabled you have write `if(!isDisabled)`. But when reading that back you need parse this "if not is-not-enabled" and eliminate the double-negation. Which is slightly inconvenient.
Like, you don't name the variable `var/isntNotDisabled` because it's hard to know what setting this to FALSE would actually do to the machine. This is exaggeration of course, but my point is that `isDisabled` is still on the same spectrum, and only `enabled` doesn't suffer from this too-many-netations problem.

Reply:
Yes, I understand now. Its not a trivial distinction, and makes for cleaner code.